### ISSUE TITLE
[feature request] Remember installed snaps sort option

### ISSUE THREAD

**Issue by fthx** [ubuntu/app-center#1631]
Hi,

This one:

![image](https://github.com/ubuntu/app-center/assets/23138504/3cf40878-6aac-4248-9573-3d3641500cd4)

Should imo be saved from last choice.

**Comment by lukas-varga** [comment]
Agree, could be nice to remember sorting option, since for me it makes sense to display the newest apps first and not in alphabetical order ;)
